packages/cli/test-helpers/index.ts

```diff
@@ -0,0 +1,38 @@
+import { PublicKey } from '@solana/web3.js';
+
+import { SolanaProvider } from '@saberhq/solana-contrib';
+import { AnchorProvider } from '@project-serum/anchor';
+import { TokadaptSDK } from '@marinade.finance/tokadapt-sdk';
+import { TokadaptHelper } from '@marinade.finance/tokadapt-sdk/test-helpers/tokadapt';
+
+import {
+  shellMatchers as untyped,
+  createTempFileKeypair,
+} from '@marinade.finance/solana-test-utils';
+
+const shellMatchers = untyped as () => void;
+export { shellMatchers };
+
+export const initSDK = () => {
+  const anchorProvider = AnchorProvider.env();
+  const sdk = new TokadaptSDK({
+    provider: SolanaProvider.init(anchorProvider),
+  });
+  return sdk;
+};
+
+export const createTokadapt = async (sdk: TokadaptSDK, admin?: PublicKey) => {
```

Comment:
I misunderstood usage of this - have renamed it to be clear this is a file wallet tokadapt create helper - for other cases I think it makes sense to just use TokadaptHelper.create directly ?

Reply:
The basic idea of using keypair here is grinding for an excellent account name. Otherwise, you are just skipping the keypair parameter for creation (it makes a random key) and using generated pubkey for all other commands

Reply:
updated to use generated key